src/Bicep.Core.Samples/Files/InvalidParameters_LF/main.diagnostics.bicep

```diff
@@ -50,19 +50,17 @@ param partialType str
 
 param malformedType 44
 //@[06:019) [no-unused-params (Warning)] Parameter "malformedType" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |malformedType|
-//@[20:022) [BCP283 (Error)] Using a literal value as a type requires enabling EXPERIMENTAL feature "UserDefinedTypes". (CodeDescription: none) |44|
 
 // malformed type but type check should still happen
 param malformedType2 44 = f
 //@[06:020) [no-unused-params (Warning)] Parameter "malformedType2" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |malformedType2|
-//@[21:023) [BCP283 (Error)] Using a literal value as a type requires enabling EXPERIMENTAL feature "UserDefinedTypes". (CodeDescription: none) |44|
+//@[26:027) [BCP057 (Error)] The name "f" does not exist in the current context. (CodeDescription: none) |f|
```

Comment:
I was a bit surprised that this diagnostic hadn't been surfaced in the previous baseline version, but apparently [param default values are not validated if there is an error encountered in the param type clause](https://github.com/Azure/bicep/blob/b89f93150c18c4e964906826e08f42eaae8a317f/src/Bicep.Core/TypeSystem/TypeAssignmentVisitor.cs#L422-L425)